src/main/java/homeTry/member/service/MemberService.java

```diff
@@ -2,54 +2,51 @@
 
 
 import homeTry.exerciseList.service.ExerciseHistoryService;
-import homeTry.member.dto.ChangeNicknameDTO;
 import homeTry.member.dto.MemberDTO;
-import homeTry.member.dto.MyPageDTO;
+import homeTry.member.dto.request.ChangeNicknameRequest;
+import homeTry.member.dto.response.MyPageResponse;
 import homeTry.member.exception.badRequestException.LoginFailedException;
 import homeTry.member.exception.badRequestException.MemberNotFoundException;
 import homeTry.member.exception.badRequestException.RegisterEmailConflictException;
-import homeTry.member.exception.internalServerException.UniqueKeyViolatonException;
 import homeTry.member.model.entity.Member;
 import homeTry.member.model.vo.Email;
 import homeTry.member.model.vo.Nickname;
 import homeTry.member.repository.MemberRepository;
-import java.time.Duration;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.time.Duration;
+
 @Service
 public class MemberService {
+
     private final ExerciseHistoryService exerciseHistoryService;
     private final MemberRepository memberRepository;
 
-    public MemberService(ExerciseHistoryService exerciseHistoryService, MemberRepository memberRepository) {
+    public MemberService(ExerciseHistoryService exerciseHistoryService,
+                         MemberRepository memberRepository) {
         this.exerciseHistoryService = exerciseHistoryService;
-        this.memberRepository = memberRepository; }
+        this.memberRepository = memberRepository;
+    }
 
     @Transactional(readOnly = true)
     public Long login(MemberDTO memberDTO) {
-        Long countByEmail = memberRepository.countByEmail(new Email(memberDTO.email()));
-
-        if (countByEmail < 1) { throw new LoginFailedException(); }
-        if (countByEmail > 1) { throw new UniqueKeyViolatonException(); }
-
-        // 로그인 성공
-        return memberRepository.findByEmail(new Email(memberDTO.email())).get().getId();
+        return memberRepository.findByEmail(new Email(memberDTO.email())).orElseThrow(LoginFailedException::new).getId();
```

Comment:
```suggestion
        return memberRepository.findByEmail(new Email(memberDTO.email()))
                               .orElseThrow(LoginFailedException::new)
                               .getId();
```

처럼 적절하게 띄워쓰기 해주시면 가독성이 더 좋아질 것 같아요~